packages/ia-zendesk-help-widget/test/zendesk-help-widget.test.js

```diff
@@ -0,0 +1,26 @@
+import {
+  html,
+  fixture,
+  expect,
+  oneEvent
+} from '@open-wc/testing';
+import '../src/ia-zendesk-help-widget';
+
+const component = (properties = {
+  widgetSrc: 'https://static.zdassets.com/ekr/snippet.js?key=',
+  widgetKey: '685f6dc4-48c5-411f-8463-cc6dd50abe2d',
```

Comment:
Odd. The `widgetKey` property is camel case here, but is not in the component definition. How are these tests passing?

Reply:
Changed these